src/main/java/com/marginallyclever/makelangelo/plotter/plotterControls/JogInterface.java

```diff
@@ -82,32 +85,35 @@ public void findHome() {
 	
 	private JToolBar getToolBar() {
 		JToolBar bar = new JToolBar();
+		bar.setFloatable(false);
+		JButton home    = new ButtonIcon("JogInterface.Home", "/images/house.png");
+		JButton penUp    = new ButtonIcon("JogInterface.PenUp", "/images/arrow_up.png");
+		JButton penDown  = new ButtonIcon("JogInterface.PenDown", "/images/arrow_down.png");
+		toggleEngageMotor = new ButtonIcon("JogInterface.DisengageMotors", "/images/lock_open.png");
 
-		JButton penUp    = new JButton(Translator.get("JogInterface.PenUp"));
-		JButton penDown  = new JButton(Translator.get("JogInterface.PenDown"));
-		JButton findHome = new JButton(Translator.get("JogInterface.FindHome"));
-		toggleEngageMotor = new JButton(Translator.get("JogInterface.DisengageMotors"));
-
-		bar.add(findHome);
+		bar.add(home);
 		bar.addSeparator();
 		bar.add(penUp);
 		bar.add(penDown);
 		bar.addSeparator();
 		bar.add(toggleEngageMotor);
-		
-		penUp.addActionListener((e)-> myPlotter.raisePen());
-		penDown.addActionListener((e)-> myPlotter.lowerPen());
-		findHome.addActionListener((e)-> myPlotter.findHome());
-		toggleEngageMotor.addActionListener((e)-> onToggleEngageMotorAction());
+
+		home.addActionListener(e -> myPlotter.findHome());
```

Comment:
there used to be a "goHome" button.  keeping it called "findHome" is less ambiguous.

Reply:
Ok renamed